corehq/messaging/scheduling/forms.py

```diff
@@ -2566,12 +2555,24 @@ def clean_user_data_property_json(self):
         if not value:
             raise ValidationError(_("This field is required."))
 
+        def json_error():
+            return ValidationError(_("Invalid JSON value: Needs to be and object with valid "
```

Comment:
typo? "Needs to be and object with valid properties" should be "Needs to be an object..."